### ISSUE TITLE
emesene for Windows glyph issue

### ISSUE THREAD

**Issue by ghost** [emesene/emesene#1589]
At least two instances of this problem occured, where default fonts ("Sans" ?) used by emesene for Windows don't seem to exist for me. The timestamp thing that appears on messages (I think that's what it is, I can't read it because the glyphs are screwed up) is the only thing left that has this issue, after I change my text font.

So, I was able to partially fix it.

By the way, in the font-chooser, the "Preview text!" is readable; however, the applied text is not.

I hope I've explained this properly.

**Comment by ghost** [comment]
To clarify: initially, I wasn't able to read either the black colored message content or the (Green, Yellow, Orange? -- I'm color blind. One of those colors) glyphs, which I suspect to be a time stamp (like: "This message was posted at date/time etc")

I fixed the message content issue  by changing the font from "Sans" to "Arial"

**Comment by c10ud** [comment]
thanks for your interest but emesene, due to its msn-centered nature, is at
the moment abandoned (the msn service has been shutdown for general people)

2014-07-09 15:29 GMT+02:00 Pie Man notifications@github.com:

**Comment by ghost** [comment]
How come I can still chat with my MSN buddies using emesene?

**Comment by dequis** [comment]
Because @c10ud lies, it wasn't really shut down, but he never bothered checking that everything still works just fine.

He's right in the part where he says the project is abandoned, though.

**Comment by c10ud** [comment]
as explained elsewhere, the service is still active.
however most users are unable to still use it since the official client
forced the upgrade to skype.

i believe some areas of the world and unofficial clients can still connect
without problems

2014-07-09 15:55 GMT+02:00 Pie Man notifications@github.com:

**Comment by dequis** [comment]
Well yeah, kinda. The official client forces update to skype, but you can still use msn to talk to people who use skype with 'microsoft accounts'.

**Comment by ghost** [comment]
That is correct! My friend said he always uses Skype now. I can't stand Skype.
